changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 26th May 2025
+* Extended [product classifications](../operations/products.md#product-classifications) with `Fee`.

```

Comment:
The preferred formatting for changelog currently is to start with operation name:

```suggestion
* [Get all products](products.md#get-all-products):
  * Extended [product classifications](../operations/products.md#product-classifications) with `Fee` property.
```

Reply:
Thanks @jonas-paul ! Should I mention all affected operations if there are?

Reply:
(Just noticed there's none, so just consider this question for future 😄)